index.html

```diff
@@ -9,7 +9,7 @@
     <script type="module">
         import Chatbot from './flowise-embed/dist/web.js';
         Chatbot.initFull({
-          chatflowid: '162459a5-b621-464d-9dc9-ae59ccc1077c',
+          chatflowid: '77a8ed32-b99a-4739-939e-e8a02d0f3d4e',
```

Comment:
Esse código ainda não corresponde ao fluxo de produção. Se fizermos o merge dessa forma, passaremos a apontar para o fluxo errado - precisamos atualizar o que for necessário no fluxo de produção antes, também.
Temos que validar as alterações no fluxo antes de fazer o merge.